code/modules/bitrunning/objects/gimmick_disks/sports_disk.dm

```diff
@@ -0,0 +1,142 @@
+
+/**
+ * Bitrunning gimmick loadouts themed around sports.
+ * Mostly for fun, have niche or little advantages.
+ */
+/obj/item/bitrunning_disk/gimmick/sports
+	name = "bitrunning gimmick: sports"
+	selectable_loadouts = list(
+		/datum/bitrunning_gimmick/boxer,
+		/datum/bitrunning_gimmick/skater,
+		/datum/bitrunning_gimmick/archer,
+		/datum/bitrunning_gimmick/fisher,
+		/datum/bitrunning_gimmick/gamer,
+	)
+
+
+/datum/bitrunning_gimmick/boxer
+	name = "Boxer"
+
+	granted_items = list(
+		/obj/item/clothing/gloves/boxing/evil,
+		/obj/item/clothing/under/shorts/red,
+		/obj/item/reagent_containers/condiment/protein,
+		/obj/item/reagent_containers/cup/glass/drinkingglass/filled/protein_blend,
+	)
+
+/obj/item/reagent_containers/cup/glass/drinkingglass/filled/protein_blend
+	name = "Protein Blend"
+	list_reagents = list(/datum/reagent/consumable/ethanol/protein_blend = 50)
+
+
+/datum/bitrunning_gimmick/skater
+	name = "Skater"
+
+	granted_items = list(
+		/obj/item/clothing/shoes/wheelys,
+		/obj/item/melee/skateboard,
+		/obj/item/clothing/suit/costume/wellworn_shirt/graphic,
+		/obj/item/clothing/head/soft/black,
+		/obj/item/clothing/shoes/sneakers/black,
+		/obj/item/storage/cans/sixenergydrink,
+	)
+
+/obj/item/storage/cans/sixenergydrink
+	name = "energy drink bottle ring"
+	desc = "Holds six energy drink cans. Remember to recycle when you're done!"
+
+	/// Pool of gamer drinks tm we may add from
+	var/list/gamer_drink_options = list(
```

Comment:
similar

Reply:
as above- unchanged name is making me suspicious